README.md

```diff
@@ -12,12 +12,12 @@ The OSP Director Operator creates a set of Custom Resource Definitions on top of
 
 Hardware Provisioning CRDs
 --------------------------
-- openstacknetattachment: manages NodeNetworkConfigurationPolicy and NodeSriovConfigurationPolicy used to attach networks to virtual machines
+- openstacknetattachment: (internal) manages NodeNetworkConfigurationPolicy and NodeSriovConfigurationPolicy used to attach networks to virtual machines
 - openstacknetconfig: high level CRD to specify openstacknetattachments and openstacknets to describe the full network configuration. The set of reserved IP/MAC addresses per node are reflected in the status.
 - openstackbaremetalset: create sets of baremetal hosts for a specific TripleO role (Compute, Storage, etc.)
 - openstackcontrolplane: A CRD used to create the OpenStack control plane and manage associated openstackvmsets
 - openstacknet: Create networks which are used to assign IPs to the vmset and baremetalset resources below
```

Comment:
should also add internal tag?

Reply:
sure, I can mark openstacknet (internal)